packages/vite/src/node/server/http.ts

```diff
@@ -133,3 +133,29 @@ async function createCertificate() {
   })
   return pems.private + pems.cert
 }
+
+async function getCertificate(config: ResolvedConfig) {
+  if (!config.cacheDir) return await createCertificate()
+
+  const cachePath = path.join(config.cacheDir, '_cert.pem')
+
+  try {
+    const [stat, content] = await Promise.all([
+      fsp.stat(cachePath),
+      fsp.readFile(cachePath, 'utf8')
+    ])
+
+    if (Date.now() - stat.ctime.valueOf() > 30 * 24 * 60 * 60 * 1000) {
+      throw new Error('cache is outdated.')
+    }
```

Comment:
- How this this timeout been picked? Should we make it configurable?
- Not sure if we should throw an error here

Reply:
- There is a [30 days hard code](https://github.com/vitejs/vite/blob/c69f83615292953d40f07b1178d1ed1d72abe695/packages/vite/src/node/server/http.ts#L75).I don't think it worth to add anoth options. see: https://github.com/vitejs/vite/blob/main/.github/contributing.md#think-before-adding-yet-another-option
- It will be catch blow.

Reply:
@antfu Is this explanation sufficient? Do I need to make any changes?